drivers/dai/intel/ssp/Kconfig.ssp

```diff
@@ -15,6 +15,7 @@ config DAI_INTEL_SSP
 
 config DAI_SSP_HAS_POWER_CONTROL
 	bool "DAI ssp pm_runtime en/dis ssp power"
+	default y if SOC_SERIES_INTEL_ACE1X
```

Comment:
was this renamed only to SOC_SERIES_INTEL_ACE, so not the 1X? I might remember incorrectly also...

Reply:
I can change to SOC_SERIES_INTEL_ACE, doesnt matter for now